recipes/termcap/all/patches/0002-macOS.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/tparam.c b/tparam.c
+index 5a9809a..d967e6d 100644
+--- a/tparam.c
++++ b/tparam.c
+@@ -22,8 +22,9 @@ Boston, MA 02111-1307, USA.  */
+ #endif
+ 
+ #ifdef emacs
+-#include "lisp.h"		/* for xmalloc */
+-#else
++#include <lisp.h>   /* xmalloc is here */
++
++#else /* not emacs */
+ 
+ #ifdef STDC_HEADERS
+ #include <stdlib.h>
```

Comment:
there is no change here.

Reply:
`"` and `<>` are equivalent, they only change priority of include folders scan. The only important part in this patch seems to be

```c++
#ifdef HAVE_UNISTD_H
#include <unistd.h>
#endif
``` 

So if you could update your patch to keep only this part, it would be great.

Reply:
already done.

Reply:
Is that ok with you @SpaceIm ?

Reply:
👍